int32 node_id
---
float64 x
float64 y